packages/protocol/contracts/common/OwnerUUPSUpgradable.sol

```diff
@@ -25,7 +25,12 @@ abstract contract OwnerUUPSUpgradable is UUPSUpgradeable, OwnableUpgradeable {
     uint8 private constant _FALSE = 1;
     uint8 private constant _TRUE = 2;
 
-    uint8 private _reentry; // slot 1
+    // The slot in transient storage of the reentry lock
+    // This is the keccak256 hash of "ownerUUPS.reentry_slot"
+    bytes32 private constant _REENTRY_SLOT =
+        0xa5054f728453d3dbe953bdc43e4d0cb97e662ea32d7958190f3dc2da31d9721a;
```

Comment:
Also use transient storage for the reentry lock. Same explanation as for the bridge context how this helps.